pypuppetdb/types.py

```diff
@@ -208,6 +221,25 @@ def __str__(self):
     def __unicode__(self):
         return self.__string
 
+    def metrics_dict(self):
```

Comment:
Should have looked at the entire context.  I thought this was getting returned instead of the metrics array.

Could make a property that takes 'category' and 'name' and returns the value.  This would use the`metrics_to_dict`
to turn the data from the array to the dict and then just keep it around.